CocoaLumberjack.podspec

```diff
@@ -24,22 +24,16 @@ Pod::Spec.new do |s|
   s.watchos.deployment_target = '2.0'
   s.tvos.deployment_target    = '9.0'
 
-  s.default_subspecs = 'Default', 'Extensions'
+  s.module_map = 'Framework/Lumberjack/CocoaLumberjack.modulemap'
+  s.default_subspecs = 'Default'
```

Comment:
@rivera-ernesto 

> I think probably you had better change the modulemap of Lumberjack.xcodeproj a little.

I checked deeply and **actually** each target's headers and modulemap. As a result  I found no need change... Sorry for confusing you...
But if I find another issue or come to better solution, I'd like to discuss or make a PR.
Thanks again!!